docs/guides/auth.md

```diff
@@ -175,26 +175,41 @@ var scope = 'https://www.googleapis.com/auth/grpc-testing';
 
 ```csharp
 // Base case - No encryption/authentication
```

Comment:
Actually, it might look nice if we had 3 tabs  for the auth example:
by default you would see Auth using JWT (the recommended approach) and you would be able to switch to  OAuth2 auth or single RPC.

But let's leave this improvement for later.